### DISCUSSION TITLE
Unable to use the package on Linux (Ubuntu) running .Net Core 5.0

### DISCUSSION THREAD

**Discussion by red4cape** [vvvv/SVG#874] [Q&A] [answered]
I have been getting the below error while trying to start my application on Ubuntu 20.04, with .Net 5.0 installed.

An assembly specified in the application dependencies manifest (Entry.deps.json) was not found: package: 'System.Drawing.Common', version: '5.0.2' path: 'runtimes/unix/lib/netcoreapp3.0/System.Drawing.Common.dll'

The project builds and runs without any errors on Mac OSx... However when I publish the Release to Ubuntu, I get the above error.

Note: I am using package Svg version 3.2.3, which is dependent on System.Drawing.Common.

I have already tried adding "System.Drawing.Common" nuget to the project but no luck. Any help is appreciated.

**Comment by red4cape** [comment, marked as answer]
Added a symbolic link to "/usr/share/dotnet/shared/Microsoft.AspNetCore.App/5.0.5" from "runtimes/unix/lib/netcoreapp3.0".
That fixed the problem. Good enough for now to unblock me.

**Comment by mrbean-bremen** [comment]
I converted this into a Q/A as it might help others with the same problem.
